frontend/src/metabase/components/Select.jsx

```diff
@@ -80,7 +80,7 @@ class BrowserSelect extends Component {
                 ref="popover"
                 className={className}
                 triggerElement={
-                    <div className={"flex align-center " + (!value ? " text-grey-3" : "")}>
+                    <div className={"flex align-center px2 py1 " + (!value ? " text-grey-3" : "")}>
```

Comment:
Gotcha. What's your thinking on the best way to prevent that? It seems like the issue we ran into with the GSG was that it passed the padding down to the popover component but this was the first instance of that I've seen.